src_bicep/template/network/main.bicep

```diff
@@ -0,0 +1,32 @@
+param name string
+param location string
+param addressPrefix string
+param addressPrefixSubnetId string
+
+resource virtualNetwork 'Microsoft.Network/virtualNetworks@2023-05-01' = {
+  name: name
+  location: location
+  properties: {
+    addressSpace: {
+      addressPrefixes: [
+        addressPrefix
+      ]
+    }
+    subnets: [
+      {
+        name: 'default'
+        properties: {
+          addressPrefix: addressPrefixSubnetId
```

Comment:
`addressPrefixSubnetId` is not a correct name here. It's better to be just `addressPrefix`